installer/PowerToysSetup/Product.wxs

```diff
@@ -18,7 +18,7 @@
 
     <Upgrade Id="42B84BF7-5FBF-473B-9C8B-049DC16F7708">
       <UpgradeVersion
-         Minimum="0.11.0" Maximum="0.14.1"
+         Minimum="0.11.0" Maximum="$(var.PreviousVersion)"
```

Comment:
What if we put the current version as the Maximum, so we avoid having to update the `PreviousVersion` in Version.props?
Would it break when we run the msi to change/remove the current installation?

Reply:
BTW I've just realized we released 0.15.1 without updating 0.14.1 to 0.15.0!

Reply:
Can you give it a try, install `0.15.0` and then run the `0.15.1` msi, what happens?

Reply:
Yes, I've also noticed that. Luckily, 0.15.0 wasn't available for a long time. I've also tried clean installing 0.15.0 -> 0.15.2. No matter if you install them to a same or different dirs, 0.15.0 is removed, but no update dialog is shown. 

So, if we set `PreviousVersion` to the current, we will be uninstalling silently the current version (relying on the internal wix logic described above) and installing it again from scratch w/o seeing the update dialog.

Reply:
> So, if we set PreviousVersion to the current, we will be uninstalling silently the current version (relying on the internal wix logic described above) and installing it again from scratch w/o seeing the update dialog.

So it won't show the Modify/Remove options? That is not acceptable.
Can we use something like `Maximum<"$(var.Version)"`?

Reply:
sure